about.html

```diff
@@ -0,0 +1,148 @@
+                        <a class="place-paragraph email-about-ask">Altinbasuni@gmail.com</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        
+        <div class="second-about-page">
+            <h2 class="main-program main-program2">ALtinbas 2020 logo</h2>
+            <hr>
+            <p class="about-p-logo"> The Altinbas Logo was Desigend in 7/june/2020</p>
+            <div class="about-logo">
+                <img class="about-logo-img" src="images/altinlogo.jpg" alt="altinbas-logo">
+            </div>
+            <hr class="about-second-page-hr">
+        </div>
+
+        <div class="third-about-page">
+            <h2 class="third-about-confreneces">See the past Altinbas Confreneces</h2>
+            <hr>
+            <p class="about-p-logo">Take a look at the last two Confreneces at altinbas university  </p>
+            <div class="about-desktop-continer">
+                <div class="about-conf-continer1">
+                        <img class="about-first-con-img" src="images/confrenece.jpeg" alt="conf-img">
+                    <div class="about-year-div">
+                        <p class="about-conf-year">2019</p>
+                    </div>
+                    <div class="about-descrption-div">
+                        <p class="about-conf-descrption">Conference on students life in altinbas</p>
+                    </div>
+                </div>
+
+                <div class="about-conf-continer1 about-conf-continer2 ">
+                    <img class="about-first-con-img" src="images/conf2.jpeg" alt="conf-img">
+                    <div class="about-year-div">
+                        <p class="about-conf-year">2020</p>
+                    </div>
+                    <div class="about-descrption-div">
+                        <p class="about-conf-descrption">Conference on students life in altinbas</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="mobile-fourth-section">
+            <h2 class="main-program">Partners</h2>
+            <hr>
+            <div class="desktop-continer">
+                <img class="google-img" src="images/google.png" alt="google-img">
+                <img class="firat-img" src="images/firat.png" alt="firat-img">
+                <img class="microsoft-img" src="images/microsoft.png" alt="microsoft-img">
+                <img class="izmer-img" src="images/izmer.png" alt="izmer-img">
+                <img class="aydin-img" src="images/aydin.png" alt="aydin-img">
+            </div>
+        </div>
+
+        <div class="mobile-fifth-section">
+            <img class="desktop-nav-logo altin-bas-logo" src="images/altinlogo.jpg" alt="altinbas-logo">
+            <p class="paragraph-fourth-section">2020 Altinbas University Confrenece Some Rights Reserved</p>
+        </div>
```

Comment:
- [ ] Please make the partner section disappear and change the background to dark on the `about` page to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/175128378-526ad2de-d9ad-459e-bc05-6d09fec5f1c5.png) | ![image](https://user-images.githubusercontent.com/24703171/174901355-44c5fe0e-99aa-4d13-a7a7-c67bf1558a5e.png) |

Reply:
done